webpack.server.config.js

```diff
@@ -1,7 +1,19 @@
 const path = require('path');
 const webpack = require('webpack');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 module.exports = {
+  mode: 'production',
```

Comment:
This is not really needed for Node applications. It doesn't really matter for the server if it's minified or not as it's only run on the server-side and not send to a client over the wire. Same goes for statically pre-render the demo app in a build step. Minifying the server or the prerender script doesn't have a real benefit.

Reply:
OK! I wasn't 100% sure 😄 
Thanks for the input (again)! I will update accordingly.

Reply:
Thank you very much for your effort on this one. I am really glad that you take care of this and dive into it. Sorry for all the comments and back and forth.

Reply:
No problem. I learned a lot, so thank you too!